### DISCUSSION TITLE
Streetmerchant don't start on CentOS8

### DISCUSSION THREAD

**Discussion by fransyk** [jef/streetmerchant#1947] [Help]
Hi,
I followed the installation steps streetmerchant with no error. When I start the software with command npm run start I have this error:

[7:10:57] error :: ✖ something bad happened, resetting streetmerchant in 5 seconds Failed to launch the browser process!
[29177:29177:0213/071057.496469:ERROR:zygote_host_impl_linux.cc(90)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.

TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md
 {
  "stack": "Error: Failed to launch the browser process!\n[29177:29177:0213/071057.496469:ERROR:zygote_host_impl_linux.cc(90)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.\n\n\nTROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md\n\n    at onClose (/root/streetmerchant/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:193:20)\n    at Interface.<anonymous> (/root/streetmerchant/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:183:68)\n    at Interface.emit (events.js:327:22)\n    at Interface.close (readline.js:424:8)\n    at Socket.onend (readline.js:202:10)\n    at Socket.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)"

I launch the software with root account

**Comment by Caporall** [comment]
try changing setting in dotenv file


BROWSER_TRUSTED | Skip Chromium Sandbox | Useful for containerized environments, default: false
